integration-tests/jpa/src/test/java/io/quarkus/it/jpa/preferredhibernatetypesoverride/OverriddenPreferredTypesResourceIT.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.it.jpa.preferredhibernatetypesoverride;
+
+import io.quarkus.it.jpa.defaultcatalogandschema.DefaultSchemaTest;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import io.quarkus.test.junit.TestProfile;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+
+@QuarkusTest
+@TestProfile(OverriddenPreferredTypesResourceIT.OverriddenPreferredTypesSchemaTestProfile.class)
+class OverriddenPreferredTypesResourceIT {
+    public static class OverriddenPreferredTypesSchemaTestProfile implements QuarkusTestProfile {
+        @Override
+        public Map<String, String> getConfigOverrides() {
+            return Map.of(
+                    "quarkus.hibernate-orm.database.default-schema", "SCHEMA1",
+                    "quarkus.hibernate-orm.sql-load-script", "no-file",
+                    "quarkus.hibernate-orm.mapping.duration.preferred-jdbc-type", "INTERVAL_SECOND",
+                    "quarkus.hibernate-orm.mapping.instant.preferred-jdbc-type", "INSTANT",
+                    "quarkus.hibernate-orm.mapping.boolean.preferred-jdbc-type", "BIT",
+                    "quarkus.hibernate-orm.mapping.uuid.preferred-jdbc-type", "CHAR"
```

Comment:
These are build-time properties, so I'm not sure they can be mentioned in a test profile for native (graalVM) tests... the properties cannot be set after native compilation.

If native tests fail, you will know why :) And in that case I'd suggest simply defining a separate (named) persistence unit where you can set all the properties you want. We've done something like this in other ITs for other features, e.g. there:

https://github.com/quarkusio/quarkus/blob/3e0bad96094be2225e3b3696ad0170f25aea368d/integration-tests/jpa-mariadb/src/main/resources/application.properties#L9-L16